addons/account/models/template_generic_coa.py

```diff
@@ -48,3 +48,13 @@ def _get_generic_coa_res_company(self):
                 'income_account_id': 'income',
             }
         }
+
+    @template(model='res.company')
+    def _get_res_company(self, template_code):
+        data = super()._get_res_company(template_code)
+        if template_code == 'generic_coa':
+            data[self.env.company.id].update({
+                'account_stock_journal_id': 'inventory_valuation',
```

Comment:
Shouldn't the expense account and the variation account be set on the generic CoA as well?